manifests/v/Viber/Viber/20.1.0.10/Viber.Viber.installer.yaml

```
@@ -0,0 +1,32 @@
+# Created with YamlCreate.ps1 v2.2.6 $debug=AUSU.CRLF.5-1-19041-2673.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Viber.Viber
+PackageVersion: 20.1.0.10
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+Scope: user
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://download.cdn.viber.com/desktop/windows/ViberSetup.msi
+  InstallerSha256: BA32BE7C21D5C390E8FE6A8D7A35087DF9569D81A8A436040D57641FD8C4AD7F
+  ProductCode: '{C83A0D5F-70D2-4A6E-9328-1FC8A552530B}'
+  AppsAndFeaturesEntries:
+    - Publisher: '2010-2023 Viber Media S.a.r.l'
+- Architecture: x64
+  InstallerType: burn
```

```suggestion
  InstallerType: burn
  Scope: user
```
Primary ARP entry is at HKCU